src/config/.config/dotnet-tools.json

```diff
@@ -3,7 +3,7 @@
   "isRoot": true,
   "tools": {
     "csharpier": {
-      "version": "0.30.0",
+      "version": "0.30.1",
```

Comment:
Fix a problem (CSharpier could not be set up properly so formatting is not currently supported) with CSharpier in VS 2022